src/anjaliS/PrintSameNumberFromArray.java

```diff
@@ -0,0 +1,21 @@
+package anjaliS;
+
+public class PrintSameNumberFromArray {
+	void displaySameNum(int[] arr1, int[] arr2) {
+		for (int i = 0; i < arr1.length; i++) {
+			for (int j = 0; j < arr2.length; j++) {
+				if (arr2[j] == arr1[i])
+					System.out.println(+arr1[i]);
```

Comment:
Just one small change, you should put break, as you get your match then why we need to iterate on second array right